addons/mail/tools/discuss.py

```diff
@@ -49,17 +49,18 @@ def get_twilio_credentials(env) -> (str, str):
 
 def get_sfu_url(env) -> str | None:
     sfu_url = env['ir.config_parameter'].sudo().get_param("mail.sfu_server_url")
-    if not sfu_url:
+    is_saas = "saas" in release.version
```

Comment:
There shouldn't be a saas fork here, saas overrides the whole function to "inject" their own credentials (I think, only if super does not return a key). The env is for SH.

(same of the other function)

Reply:
since aju wants to keep the setting, I think that you could do the check for `use_twilio_rtc_servers` (and the new setting for SFU), in their respective URL getters:

if the setting is off, sfu_url is set to `None` here, which will disable the feature but still allow saas to override the method and return a URL that will enable the feature.